Controller/HelperController.php

```diff
@@ -112,7 +112,13 @@ public function appendFormFieldElementAction(Request $request)
 
         $extension = $this->twig->getExtension('Symfony\Bridge\Twig\Extension\FormExtension');
         $extension->initRuntime($this->twig);
-        $extension->renderer->setTheme($view, $admin->getFormTheme());
+
+        if ($this->has('twig.form.renderer')) {
```

Comment:
`HelperController` does not extend the standard sf controller, hence the error you get in Travis